libraries/node-core-library/src/LockFile.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as fsx from 'fs-extra';
+import * as path from 'path';
+import * as child_process from 'child_process';
+
+export function getProcessStartTime(pid: string): string | undefined {
```

Comment:
>pid [](start = 36, length = 3)

Why is `pid` a string and not an integer?

Reply:
I guess it could be easier, but it is easier to .toString() the pid from process.pid than it is to parseInt(pid, 10) from the old lock file.

---
In reply to: [165796614](https://github.com/Microsoft/web-build-tools/pull/517#discussion_r165796614) [](ancestors = 165796614)